index.js

```diff
@@ -366,14 +389,17 @@ function separator (scanner) {
   } else {
     return invalidToken(scanner, ['separator'])
   }
+  node.position = { start, end: scanner.position() }
   return node
 }
 
 function invalidToken (scanner, expected) {
   if (scanner.eof()) {
     return Error(`unexpected token EOF valid tokens [${expected.join(', ')}]`)
   } else {
-    return Error(`unexpected token '${scanner.peek()}' at position ${scanner.position()} valid tokens [${expected.join(', ')}]`)
+    const pos = scanner.position()
+    const posString = `{ line: ${pos.line}, column: ${pos.column}, offset: ${pos.offset} }`
```

Comment:
We can also revert this back to `... at position ${scanner.position().offset} ...` if it's too much?

Reply:
Perhaps we could model after stack traces and just show `line:column`?